### DISCUSSION TITLE
Welcome to react-loader-spinner Discussions!

### DISCUSSION THREAD

**Discussion by mhnpd** [mhnpd/react-loader-spinner#162] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Yubraj977** [comment]
Hey Guys , There is not step by step setup process somebody can update contribute.md
